### PR TITLE
Vellum case management warning on revert

### PR DESCRIPTION
**Pull request by orangejenny**
Forked off of https://github.com/dimagi/commcare-hq/pull/12381

@emord / @kaapstorm 

@amsagoff let me know if you have better text for the warning

screen shot 2016-07-08 at 6 15 16 pm

### CODE REVIEW

File: corehq/apps/app_manager/_design/views/saved_app/map.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ function(doc){
             comment_from: doc.comment_from,
             is_released: doc.is_released,
             case_sharing: doc.case_sharing,
-            build_profiles: doc.build_profiles
+            build_profiles: doc.build_profiles,
+            vellum_case_management: !!doc.vellum_case_management
```
Comment (emord) [inline corehq/apps/app_manager/_design/views/saved_app/map.js]: This would cause a reindex on couch right? I don't see a place where you use this?

Comment (orangejenny) [inline corehq/apps/app_manager/_design/views/saved_app/map.js]: Yeah, it'll reindex the app db. Knockout uses the new property: https://github.com/dimagi/commcare-hq/pull/12387/files#diff-0e1fa007e17c990453d2c608a2de55a9R313

Comment (emord) [inline corehq/apps/app_manager/_design/views/saved_app/map.js]: Ah I see, didn't realize that was coming from a view.

I'm not to familiar on how fast reindexing the app db would be. @dannyroberts any thoughts?

Comment (dannyroberts) [inline corehq/apps/app_manager/_design/views/saved_app/map.js]: @orangejenny reached out to me about this Friday. I said (and I still believe this) that with the app db split out, this is actually a pretty quick operation (i.e. <1h; I recall it being in the minutes but I could be wrong). I wouldn't be too worried about it!

**Comment by dannyroberts** [comment]
(Not mergeable)

**Comment by orangejenny** [comment]
mergeable!

**Pull request merged**